documentation/staging/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -756,7 +756,7 @@ _Here are some of the expected WebLogic pod labels after an online update succes
      * Each WebLogic Server pod's `weblogic.introspectVersion` label will match `domain.spec.introspectVersion` after it is rolled.
      * The domain status `Available` condition will have a `Status` of `True` after the roll completes.
    * Actions required:
-     * If you want the non-dynamic changes to take effect, then restart the pod(s) with the `weblogic.configChangesPendingRestart=true` label (such as by initiating a domain roll).
+     * None. All changes will complete once the operator initiated domain roll completes.
```

Comment:
once  -> after (once – don’t use to mean after; use only to mean less than twice.)

Reply:
Fixed.